modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
The intention of this line is to actually start getty on the serial line after it's been enabled. Otherwise we'd need to manually start it or wait for a reboot.

Do you have a better proposal for how we should write this?